build/deps.mk

```diff
@@ -0,0 +1,107 @@
+# List of vendored go executables needed for make generate
+GEN_GO_BIN_DEPS = \
+	github.com/golang/mock/mockgen \
+	github.com/gogo/protobuf/protoc-gen-gogoslick \
+	github.com/uber/tchannel-go/thrift/thrift-gen \
+	golang.org/x/tools/cmd/stringer \
+	go.uber.org/thriftrw \
```

Comment:
I'm always nervous when it comes to installing bins from master...

Makes me wonder if we can get these things using glide and then install them directly out of `vendor/` so we can control the versions explicitly.